cmd/podman/libpodruntime/runtime.go

```diff
@@ -171,7 +171,7 @@ func getRuntime(ctx context.Context, c *cliconfig.PodmanCommand, renumber, migra
 		options = append(options, libpod.WithDefaultInfraCommand(infraCommand))
 	}
 
-	if withFDS {
+	if !withFDS {
```

Comment:
Maybe it's me pre-tea, but I've no clue what "withFDS" stands for.  This function badly needs a few code comments explaining what each parameter stands for.  All the calls to it that I can find look like:

```
./cmd/podman/libpodruntime/runtime.go:	return getRuntime(ctx, c, false, true, false, true)
```

which doesn't enlighten anything.

Reply:
I absolutely agree. Maybe we can do some code/comment cleaning soon? Ultimately, I'd like to defer doing that to another PR.

Reply:
I'll agree, but only if an issue it created for at least this function.